elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -280,12 +286,14 @@ public ElideResponse delete(
             } else {
                 doc = new JsonApiDocument();
             }
-            RequestScope requestScope = new RequestScope(doc, transaction, user, dictionary, mapper, securityMode);
+            RequestScope requestScope = new RequestScope(
+                    doc, transaction, user, dictionary, mapper, auditLogger, securityMode);
             DeleteVisitor visitor = new DeleteVisitor(requestScope);
             Supplier<Pair<Integer, JsonNode>> responder = visitor.visit(parse(path));
             requestScope.runDeferredPermissionChecks();
             transaction.flush();
             ElideResponse response = buildResponse(responder.get());
+            auditLogger.commit();
```

Comment:
What happens if these are a shared transaction? Will we call `commit()` twice? If so, does this cause any issues?